scss/components/outside-namespace/_civicrm-usermenu-wrapper.scss

```diff
@@ -0,0 +1,27 @@
+#civicrm-usermenu-wrapper {
+  background: $crm-white;
+  border-bottom: 1px solid $crm-grayblue-dark;
+  box-shadow: 0 3px 18px 0 rgba($gray-darker, 0.25);
```

Comment:
Lets save box shadow in `variables.scss`, otherwise there is risk of having inconsistent box-shadows.

Reply:
Using the value coming from `$box-shadow` variable that is defined on shoreditch.